### PR TITLE
Update info screen for statistics (EXPOSUREAPP-9837)

### PR DESCRIPTION
**Pull request by AlexanderAlferov**
## Whats new

Updated info screen for statistics

Screenshot 2021-10-12 at 17 38 38

## Jira
[Story](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-9449)
[Ticket](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-9837)

## How to test
1) Press on info button on any card
2) Check text and margins

**Note or testers**: kindly check not only changed that marked with magenta in Figma, but also full screen. There are multiple changes in margins and additional texts (that somehow were left between chairs in previous sprints). Thank you.

### CODE REVIEW

**Review by KathrinLuecke — DISMISSED** [review]
Please adapt string statistics_explanation_7_day_confirmed_new_infection_text

**Review by jurajkusnier — COMMENTED** [review]
It looks like we have many differences between Figma and our strings.

device-2021-10-14-152333

**Review by CV113 — DISMISSED** [review]
UA approved

**Review by CV113 — APPROVED** [review]
UA approved

**Review by CV113 — APPROVED** [review]
UA approved

File: Corona-Warn-App/src/main/res/layout/fragment_statistics_explanation.xml [hunk 1]
```diff
@@ -217,7 +272,7 @@
                     android:layout_width="match_parent"
                     android:layout_height="wrap_content"
                     android:layout_marginStart="@dimen/guideline_start"
-                    android:layout_marginTop="36dp"
+                    android:layout_marginTop="32dp"
```
Comment (jurajkusnier) [inline Corona-Warn-App/src/main/res/layout/fragment_statistics_explanation.xml:275]: Could you use dimension values such as `spacing_medium`/`spacing_normal`/etc. when you already updating values?

Comment (AlexanderAlferov) [reply Corona-Warn-App/src/main/res/layout/fragment_statistics_explanation.xml:275]: afaik last year we decided not to be hardly attached to dimens.xml in xml layouts

**Review by jurajkusnier — APPROVED** [review]

**Review by SamuraiKek — APPROVED** [review]
Looks good now 👍

**Pull request merged**